mm2src/derives/enum_from/src/from_trait.rs

```diff
@@ -8,13 +8,21 @@ use syn::{NestedMeta, Variant};
 
 impl CompileError {
     /// This error constructor is involved to be used on `EnumFromTrait` macro.
-    fn expected_trait_method_path() -> CompileError {
+    pub(crate) fn expected_trait_method_path() -> CompileError {
         CompileError(format!(
             "'{}' attribute must consist of two parts: 'Trait::method'. For example, #[{}(Default::default)]",
             MacroAttr::FromTrait,
             MacroAttr::FromTrait
         ))
     }
+
+    /// This error constructor is involved to be used on `EnumFromTrait` macro.
+    pub(crate) fn expected_literal_inner() -> CompileError {
```

Comment:
Please move this method to `from_stringify.rs`. It's possible to implement a structure methods at the multiple submodules.

Reply:
done